web/src/main/ui/app/components/flows-new/edit-flow/ui/edit-flow-ui.component.ts

```diff
@@ -117,6 +117,7 @@ export class EditFlowUiComponent {
     dialogRef.afterClosed().subscribe(response => {
       if (response) {
         this.deleteFlow.emit(this.flow.id);
+        this.router.navigate(['flows']);
```

Comment:
That shouldn't be done from ui implementation, please move it to logic component.